tests/security/mariadb/mariadb_ssl_server.pm

```diff
@@ -0,0 +1,55 @@
+# SUSE's openQA tests
+#
+# Copyright 2022 SUSE LLC
+# SPDX-License-Identifier: FSFAP
+#
+# Summary: Setup mariadb server enviroment and wait client to connect
+# Maintainer: Starry Wang <starry.wang@suse.com> Ben Chou <bchou@suse.com>
+# Tags: poo#109154
```

Comment:
Please add the tc#$test-case-number if have

Reply:
Thanks, added.